wasm/Cargo.toml

```diff
@@ -15,6 +15,7 @@ crate-type = ["rlib", "cdylib"]
 doctest = false
 
 [dependencies]
+futures-util = { workspace = true }
```

Comment:
It may be the web interface, but I can't see where this dep is used.

Reply:
It's here:
https://github.com/microsoft/qsharp/blob/19f5fb5cd6ec59cf8eaf7f4ac418f75866dd509c/language_service/src/lib.rs#L27